Makefile

```diff
@@ -18,3 +18,6 @@ test:
 
 lint:
 	hatch run dev-env:pre-commit run --show-diff-on-failure --color=always --all-files
+
+json_schema:
```

Comment:
This make target was mentioned in CI, but it didn't actually exist in the repo so I added it.